src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps you can add header comments here to describe what does this interface represent.
Recall from tutorial that an interface is a behaviour contract, perhaps `Represents a printable entity ......`.